testsuite/src/it/java/io/quarkus/ts/startstop/utils/Logs.java

```diff
@@ -84,13 +84,20 @@ public static void checkLog(String testClass, String testMethod, Apps app, MvnCm
                 if (error) {
                     if (isWhiteListed(app.whitelistLogLines.errs, line)) {
                         LOGGER.info(cmd.name() + " log for " + testMethod + " contains whitelisted error: `" + line + "'");
-                    } else  if (isWhiteListed(app.whitelistLogLines.platformErrs(), line)) {
+                    } else if (isWhiteListed(app.whitelistLogLines.platformErrs(), line)) {
                         LOGGER.info(cmd.name() + " log for " + testMethod + " contains platform specific whitelisted error: `" + line + "'");
                     } else {
                         offendingLines.add(line);
                     }
                 }
             }
+            
+            // Quarkus is not being gratefully shutdown in Windows when running in Dev mode.
+            // Reported by https://github.com/quarkusio/quarkus/issues/14647.
+            if (Commands.isThisWindows && offendingLines.stream().anyMatch(line -> WINDOWS_DEV_MODE_ERROR.matcher(line).matches())) {
+            	offendingLines.clear();
```

Comment:
this clear all the errors / warnings.

You should remove just the one related to shutdown hook not being processed properly => exit code is 1